src/current/v24.3/show-logical-replication-jobs.md

```diff
@@ -0,0 +1,86 @@
+---
+title: SHOW LOGICAL REPLICATION JOBS
+summary: The SHOW LOGICAL REPLICATION JOBS statement provides details on LDR jobs on the cluster.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Logical data replication is only supported in CockroachDB {{ site.data.products.core }} clusters.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v24.3" %} The `SHOW LOGICAL REPLICATION JOBS` statement shows details of [**logical data replication (LDR)**]({% link {{ page.version.version }}/logical-data-replication-overview.md %}) jobs on a cluster.
+
+This page is a reference for the `SHOW LOGICAL REPLICATION JOBS` SQL statement, which includes information on its parameters and possible options. For more details on monitoring LDR, refer to the [Monitor Logical Data Replication]({% link {{ page.version.version }}/logical-data-replication-monitoring.md %}) page.
+
+## Required privileges
+
+You must have one of the following to run `SHOW LOGICAL REPLICATION JOBS`:
+
+- The [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
+- The [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}).
```

Comment:
actually, neither the REPLICATION nor the CONTROL JOB privs are sufficient to run this function, at least in the little test i just ran. they need view job.

Reply:
Oh ok, I thought I had tested this too, but I'm just removing `replication` + `controljob` jic